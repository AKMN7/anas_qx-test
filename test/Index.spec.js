// import { mount } from '@vue/test-utils'
import Vue from 'vue'
import Index from '../pages/index.vue'

describe('Index', () => {
  let cmp, vm

  beforeEach(() => {
    cmp = Vue.extend(Index)

    vm = new cmp({
      data: {
        sessions: ['Test Session'],
      },
    }).$mount()
  })

  it('equals sessions to ["Test Sessions"]', () => {
    expect(vm.sessions).toEqual(['Test Session'])
  })

  it('has the expected html structure', () => {
    expect(vm.$el).toMatchSnapshot()
  })
})
