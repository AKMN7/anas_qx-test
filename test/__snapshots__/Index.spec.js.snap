// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index has the expected html structure 1`] = `<!---->`;
